src/hiero_sdk_python/transaction/transaction.py

```
@@ -129,7 +133,9 @@ def _should_retry(self, response):
             _ExecutionState: The execution state indicating what to do next
         """
         if not isinstance(response, TransactionResponseProto):
-            raise ValueError(f"Expected TransactionResponseProto but got {type(response)}")
+            raise ValueError(
+                f"Expected TransactionResponseProto but got {type(response)}"
+            )
```

Ruff (0.14.14)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)